shared/common-adapters/dumb.desktop.js

```diff
@@ -196,11 +196,36 @@ const avatarMap: DumbComponentMap<Avatar> = {
   },
 }
 
+const choiceScreenMap: DumbComponentMap<ChoiceScreen> = {
+  component: ChoiceScreen,
+  mocks: {
+    'Two Choices - Website': {
```

Comment:
:sheep: One downside to using CSS instead of props to style this is no dumb hovered screenshot. Probably not worth the effort, though!